subsys/spm/Kconfig

```diff
@@ -135,3 +138,9 @@ config SPM_NRF_SAADC_NS
 endif # IS_SPM
 
 endmenu
+
+if SPM || IS_SPM
+# Set new default.
```

Comment:
@oyvindronningstad can you write a more detailed comment here?

Reply:
Done